src/documentation/components/Wrapper.js

```diff
@@ -0,0 +1,91 @@
+const freeze = (object, property, value) => {
+  Object.defineProperty(object, property, {
+    configurable: true,
+    get() { return value; },
```

Comment:
My prettier is finding differences again on this file, e.g. here:

```
    get() {
      return value;
    },
```
Could be worth double checking that linting was done on the file just in case :)

Reply:
Yes, linting was done on the file